lib/vanity/frameworks/rails.rb

```diff
@@ -60,18 +71,15 @@ def use_vanity(symbol = nil, &block)
             # new user.id to avoid the flash of alternative B (FOAB).
             if request.get? && params[:_identity]
               @vanity_identity = params[:_identity]
-              cookies["vanity_id"] = { :value=>@vanity_identity, :expires=>1.month.from_now }
+              cookies[ Vanity.configuration.cookie_name ] = cookie.call( @vanity_identity )
```

Comment:
Another style nick pick (trying to keep the codebase consistent :grinning:): can you remove the spaces after/before closing and opening brackets/parens?